testdata/wait-epmd.ps1

```diff
@@ -0,0 +1,25 @@
+$running = $false
+[int]$count = 1
+
+if ($env:ERLANG_HOME) {
+    $epmd = Join-Path -Path $env:ERLANG_HOME 'erts-8.3\bin\epmd.exe'
```

Comment:
In appveyor.yml, we have a reference to`erlang_19.3.exe` presumubly, 19.3 is the version number of erlang we want. What's the 8.3 in `erts-8.3\bin\epmd.exe`? I am not familiar with the erlang artifacts, so this looks like a version mismatch to me.

Reply:
It's just how the internal version works. 20.2 is `erts-9.2`, I believe. Not sure why it works that way but that's how it works on Unix and Windows.